addons/tmf_loadouts/loadouts/US_USMC_2010_wdl.hpp

```diff
@@ -0,0 +1,563 @@
+		"ACE_MapTools"
+	};
+};
+
+class hatg : car {
+	displayName = "HAT Gunner";
+	backPack[] = {"CUP_B_Tow_Gun_Bag"};
+};
+
+class hatac : r {
+	displayName = "HAT Ammo Carrier";
+	backPack[] = {"CUP_B_TOW2_Tripod_Bag"};
+};
+
+class hattl : car {
+	displayName = "HAT Team Leader";
+	backPack[] = {"CUP_B_TOW2_Tripod_Bag"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class mtrg : car {
+	displayName = "Mortar Gunner";
+	secondaryWeapon[] = {"ace_csw_staticMortarCarry"};
+	backPack[] = {"B_Carryall_cbr"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrac : r {
+	displayName = "Mortar Ammo Carrier";
+	backPack[] = {"B_Carryall_cbr"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrtl : car {
+	displayName = "Mortar Team Leader";
+	secondaryWeapon[] = {"ace_csw_carryMortarBaseplate"};
+	items[] = { "MTR_GEAR" };
+	backPack[] = {"B_Carryall_cbr"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+	linkedItems[] += {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class samg : car {
+	displayName = "AA Missile Specialist";
+	secondaryWeapon[] = {"CUP_launch_FIM92Stinger"};
+};
+
+class samag : car {
+	displayName = "AA Assistant Missile Specialist";
+	secondaryWeapon[] = {"CUP_launch_FIM92Stinger"};
+	linkedItems[] += {
+		"Rangefinder",
+		"ItemGPS"
+	};
+};
+
+class sn : r {
+	displayName = "Sniper";
+	headgear[] = {"CUP_H_USMC_BOONIE_PRR_DES"};
+	uniform[] = {"CUP_U_B_USMC_FROG1_DMARPAT"};
+	goggles[] = {"default"};
+	vest[] = {"CFP_Alice_Vest2"};
+	primaryWeapon[] = {"CUP_srifle_M40A3_bipod"};
+	sidearmWeapon[] = {"CUP_hgun_M9"};
+	magazines[] = {
+		LIST_7("CUP_5Rnd_762x51_M24"),
+		LIST_2("HandGrenade"),
+		LIST_4("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class sp : r {
+	displayName = "Spotter";
+	backPack[] = {"CUP_B_AlicePack_Khaki"};
+	backpackItems[] = {
+			LIST_10("CUP_5Rnd_762x51_M24")
+	};
+	items[] += {
+		"ACE_MapTools",
+		"ACE_RangeCard",
+		"ACE_Kestrel4500",
+		"ACE_ATragMX"
+	};
+	linkedItems[] += {
+		"ACE_Yardage450",
+		"ItemGPS"
+	};
+};
+
+class vg : smg {
+	displayName = "Vehicle Gunner";
+	uniform[] = {"SP_0000_Standard_Coverall_Tan"};
+	vest[] = {"CUP_V_B_Eagle_SPC_Crew"};
+	backpack[] = {};
+	headgear[] = {"SP_CVCHelmet_Tan"};
+	magazines[] = {
+		LIST_3("CUP_30Rnd_9x19_MP5"),
+		"SmokeShellPurple"
+	};
+};
+
+class vc : vg {
+	displayName = "Vehicle Commander";
+	linkedItems[] += {
+		"Binocular",
+		"ItemGPS"
+	};
+};
+
+class vd : vg {
+	displayName = "Vehicle Driver";
+	traits[] = {"engineer"};
+	backpack[] = {"CUP_B_AssaultPack_Coyote"};
+	linkedItems[] += {"ItemGPS"};
+	backpackItems[] = {"ToolKit"};
+};
+
+class hc : smg {
+	displayName = "Helicopter Crew";
+	uniform[] = {"FIR_Fighter_Pilot_USN_Nomex4"};
+	vest[] = {"FIR_pilot_vest"};
```

Comment:
Fighter pilot gear for helicopter pilots? Haven't checked it out in game.

Reply:
Fix'd - A lot of the reference pictures seem to show USMC Rotor Wing Aviation using coveralls/flightsuits. I've replaced the crew uniforms with the tan mechanic coveralls to distinguish them between the fighter pilots.